src/i18n/de-de/index.js

```diff
@@ -0,0 +1,363 @@
+        erc1155_transfers: 'ERC1155 Transfers',
+        tokens: 'Tokens',
+        created_at_trx: 'Erstellt in Transaktion',
+        by_address: 'Durch Konto/Adresse',
+        number_used_once: 'Einmalig verwendete Nummer (nonce)',
+        native_account: 'Konto: Telos Native',
+        balance: 'Saldo',
+        view_source_prompt: 'Dieser Smart Contract wurde überprüft. Sie können den Quellcode und die Metadaten im Reiter \'contract\' ansehen',
+        account_url: '{ domain }/account/{ account }',
+        tlos_balance: '{ balance } TLOS',
+        couldnt_retreive_metadata_for_address: 'Metadata konnten nicht abgerufen werden { address }: { message }',
+        transaction_details: 'Transaktions-Details',
+        transaction_not_found: 'Nicht gefunden: { hash }',
+        general: 'Allgemein',
+        details: 'Details',
+        logs: 'Protokolle',
+        internal_txns: 'Interne Transaktionen',
+        transaction_hash: 'Transaktions Hash',
+        click_to_change_date_format: 'Klicken Sie hier, um das Datumsformat zu ändern',
+        block_number: 'Block Nummer',
+        from: 'Von',
+        to: 'Nach',
+        date: 'Datum',
+        success: 'Erfolg',
+        failure: 'Fehlgeschlagen',
+        status: 'Status',
+        balance_gwei: '{ amount } GWEI',
+        balance_tlos: '{ amount } TLOS',
+        error_message: 'Fehlermeldung',
+        contract_function: 'Smart Contract Funktionen',
+        function_parameters: 'Funktions-Parameter',
+        deployed_contract: 'Bereitgestellter Smart Contract',
+        click_to_show_in_wei: 'Anklicken zum Einblenden in wei',
+        gas_price_charged: 'Berechnete Transaktionskosten',
+        gas_fee: 'Transaktionskosten',
+        gas_limit: 'Transaktionskosten-Limit',
+        nonce: 'Nonce',
+        input: 'Eingabe',
+        output: 'Ausgabe',
+        value: 'Wert',
+    },
+    components: {
+        internal_txns: 'Interne Transaktionen',
+        n_internal_txns: '{ amount } interne transaktionen',
+        none: 'Keine',
+        verify_prompt: 'Dieser Smart Contract wurde noch nicht überprüft.  Möchten Sie den Contract/die Contracts und deren Metadaten jetzt hochladen, um die Quelle zu verifizieren?',
+        verify_contract: 'Smart Contract verifizieren',
+        search_evm_address_failed: 'Suche nach EVM-Adresse zu Telos Native Konto { accountName } fehlgeschlagen. Sie können auf wallet.telos.net eine EVM-Adresse erstellen.',
+        unknown_web3_login_type: 'Unbekannter web3 Login Typ: { provider }',
+        connect_wallet: 'Wallet verbinden',
+        view_address: 'Adresse ansehen',
+        disconnect: 'Verbindung trennen',
+        disable_wallet_extensions: 'Disable extension wallets or set Brave Wallet as default in browser wallet settings to use Brave Wallet',
+        enable_wallet_extensions: 'Enable MetaMask extension and set default to `prefer extensions` in browser wallet settings to use MetaMask wallet',
```

Comment:
These 2 lines are also new and need translation

Reply:
done